src/pip/_internal/operations/install/wheel.py

```diff
@@ -262,7 +262,10 @@ def _record_to_fs_path(record_path):
 def _fs_to_record_path(path, relative_to=None):
     # type: (text_type, Optional[text_type]) -> RecordPath
     if relative_to is not None:
-        path = os.path.relpath(path, relative_to)
+        # On Windows, do not handle relative paths if they belong to different
+        # logical disks
+        if os.path.splitdrive(path)[0] != os.path.splitdrive(relative_to)[0]:
```

Comment:
Probably want to `.lower()` the drive name to avoid catching false positives e.g. `c:` vs `C:`.

Reply:
And is the condition backwards? `relpath` fails if the drives don’t match, so this should be `==` instead.

Reply:
You are right..